js/core/templates/webidl-contiguous/typedef.html

```diff
@@ -1,2 +1,2 @@
-<span class='idlTypedef' id='{{obj.idlId}}' data-idl data-title='{{obj.name}}'>typedef<span class='idlTypedefType'>{{idlType obj
-}}</span> <span class='idlTypedefID'>{{#tryLink obj}}{{obj.name}}{{/tryLink}}</span>;</span>
\ No newline at end of file
+<span class='idlTypedef' id='{{obj.idlId}}' data-idl data-title='{{obj.name}}'>{{{trivia obj.trivia.base}}}typedef<span class='idlTypedefType'>{{idlType obj
```

Comment:
@saschanaz, is `termination` ok here? Also, I was unsure where to put `trivia.name`? See also https://github.com/w3c/webidl2.js/issues/225

Reply:
>is termination ok here?

Yep.

> I was unsure where to put trivia.name?

Probably before `<span class='idlTypedefID>`, replacing the existing space character.

Reply:
Ok, trivia for name is in!